velo/payment/tasks.py

```diff
@@ -20,3 +21,33 @@ def get_transaction_statuses():
         validate_payment(payment)
 
     return True
+
+
+@celery.task
+def process_server_response(transaction_id):
```

Comment:
Rename to process_firstdata_response. Remove Swedbank and ibank, because this function is used only for firstdata response processing.